hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/datanode/TestBPOfferService.java

```diff
@@ -1254,8 +1259,7 @@ public void testSetIsSlownode() throws Exception {
   @Test(timeout = 15000)
   public void testCommandProcessingThread() throws Exception {
     Configuration conf = new HdfsConfiguration();
-    MiniDFSCluster cluster = new MiniDFSCluster.Builder(conf).build();
-    try {
+    try (MiniDFSCluster cluster = new MiniDFSCluster.Builder(conf, baseDir.getRoot()).build()) {
```

Comment:
creating that builder with a new HdfsConfiguration is something which may make sense to factor out. certainly i'd request that in new code. What do you think?